ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -39,3 +43,11 @@ Style/NegatedIf:
 # use "raise" instead of "fail"
 Style/SignalException:
   EnforcedStyle: only_raise
+
+# do not force %r
+Style/RegexpLiteral:
+  Enabled: false
+
+# Forbid and/or usage in conditionals
+Style/AndOr:
```

Comment:
Um, I'm just not sure about `and` and `or` keywords, I'd avoid them everywhere... It's too confusing IMHO.

I'd leave the RuboCop default (`always`), see https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L127